packages/big-design/src/components/TableNext/spec.tsx

```diff
@@ -879,10 +879,8 @@ describe('expandable', () => {
     expect(onExpandedChange).not.toHaveBeenCalledWith({ 0: true, 1: true, 2: true });
   });
 
-  test('renders a cell with a helper row component in a expanded parent', async () => {
-    const HelperRow: FC<{ parentRowIndex: number }> = ({ parentRowIndex }) => {
-      return <button>View more {parentRowIndex}</button>;
-    };
+  test('when loadMoreAction exists a cell with a button is rendered', async () => {
```

Comment:
🍹 I think we've been trying to use `it` vs. `test`
```suggestion
  it('renders a "View more" button on an expandable row', async () => {
```

Reply:
I think I'm gonna keep for now `test` since this is what we are using for now in BD 😅